test/lib/Router_test.js

```diff
@@ -228,7 +228,7 @@ describe('API Router', function() {
         });
 
         describe('delete', function() {
-            it('does not attempt to validate response with schema', function() {
+            it('validates response with schema', function() {
```

Comment:
This example shows it failing schema validation. Should we note this in the description? Should we test success as well?